lib/wibox/container/radialprogressbar.lua

```diff
@@ -232,8 +245,14 @@ end
 -- @tparam number max_value
 -- @propemits true false
 
-for _, prop in ipairs {"max_value", "min_value", "border_color", "color",
-    "border_width", "paddings"} do
+--- The arc radius.
+--
```

Comment:
I think you want to add this line:

```suggestion
--
-- @DOC_wibox_container_radialprogressbar_radius_EXAMPLE@
```

This will use the image generated by `tests/examples/wibox/container/radialprogressbar/radius.lua` in the property definition in the documentation. Just like it does for instance with https://awesomewm.org/apidoc/widget_containers/wibox.container.radialprogressbar.html#paddings.